src/state/files.js

```diff
@@ -97,6 +97,10 @@ const deleteFiles = async (project, files) => {
   return files;
 };
 
+const getDocument = async (project, document) => {
+  return FileService.getDocument(project, document);
+};
```

Comment:
This method is not necessary because you can call `FilesService` directly from the visa state.